data/data/azure/vnet/main.tf

```diff
@@ -102,3 +102,16 @@ resource "azurerm_image" "cluster" {
     blob_uri = azurerm_storage_blob.rhcos_image.url
   }
 }
+
+resource "azurerm_image" "clusterv2" {
```

Comment:
I am hesitant to add another image to the resource group. What do we think the use cases are for the customer mixing V1 and V2 images? The one that I can think of would be a customer starting with V1 images and wanting to move to an instance type post-installation that only supports V2. That is a scenario that would also come up for a customer upgrading from earlier OpenShift versions where we only create the V1 image. I would expect that we would need documentation on how the user could add the V2 image manually.

Reply:
> What do we think the use cases are for the customer mixing V1 and V2 images? 

In general the use case is any time users want to create machines with an instance type that only supports one generation that is not the default image we would be creating. For example, currently we create a V1 image, so to create a machineset of V2-only instance types, we would need a second image.

> That is a scenario that would also come up for a customer upgrading from earlier OpenShift versions where we only create the V1 image. I would expect that we would need documentation on how the user could add the V2 image manually.

This is a great point, yes we would need this. Not to put too fine a point on in it, but it's not really tied to upgrades. RHCOS in 4.9 supports this now, so the image could be created manually in a 4.9 cluster without upgrading.

Reply:
@staebler did this explanation help at all? My only concern here is about usability; specifically will it be clear when users are crafting machinesets day-2. Did you have any further concerns beyond that? I have not checked pricing but I imagine this would be inconsequential.

Reply:
OK. The cost looks to be on the order of dollars per month, which is pretty inconsequential. We are not transferring data, so that is not a concern either. I am satisfied.

Reply:
Thanks for checking the cost.